.github/workflows/daily.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       run: ./runtest-sentinel
     - name: cluster tests
       run: ./runtest-cluster
+    - name: unittest
+      run: ./src/redis-server test all --accurate
```

Comment:
i was hoping to use this `accurate` feature to reduce the number of prints.
so either we remove the `--accurate` from here, or find a few of the tests that print the most, and tone them down.

Reply:
Got it.